pkg/operator/resourcesynccontroller/resourcesynccontroller.go

```diff
@@ -0,0 +1,31 @@
+package resourcesynccontroller
+
+import (
+	"github.com/openshift/cluster-kube-scheduler-operator/pkg/operator/operatorclient"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resourcesynccontroller"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	"k8s.io/client-go/kubernetes"
+)
+
+func NewResourceSyncController(
+	operatorConfigClient v1helpers.OperatorClient,
+	kubeInformersForNamespaces v1helpers.KubeInformersForNamespaces,
+	kubeClient kubernetes.Interface,
+	eventRecorder events.Recorder) (*resourcesynccontroller.ResourceSyncController, error) {
+
+	resourceSyncController := resourcesynccontroller.NewResourceSyncController(
+		operatorConfigClient,
+		kubeInformersForNamespaces,
+		kubeClient,
+		eventRecorder,
+	)
+	if err := resourceSyncController.SyncConfigMap(
```

Comment:
@ravisantoshgudimetla I'm assuming this is the wrong configmap for this operator. Are there any resources that this operator will need to sync?

Reply:
cc @deads2k ^

Reply:
At this point of time, no, I am planning to add one soon.

Reply:
Okay. Will remove this block from the PR then. Will still leave the empty resourcesynccontroller though, to at least make adding those easier in the future